src/server/es/filter.js

```diff
@@ -2,22 +2,34 @@ import { ApolloError, UserInputError } from 'apollo-server';
 import { esFieldNumericTextTypeMapping, NumericTextTypeTypeEnum } from './const';
 import config from '../config';
 
+const fromPathToNode = (esInstance, esIndex, path) => {
+  let node = esInstance.fieldTypes[esIndex];
+  if (path !== null && path !== undefined) {
+    const nodes = path.split('.');
+    nodes.forEach((n) => { node = node[n].properties; });
```

Comment:
Could we add check here, node[n] might be undefined. Throw a USER_ERROR if undefined.